stdlib/native.rb

```diff
@@ -213,6 +213,10 @@ def Native(obj)
       nil
     elsif native?(obj)
       Native::Object.new(obj)
+    elsif obj.is_a?(Array)
```

Comment:
`Array` and `Proc` not work properly, but `String`, `Float`, `Integer` work good.